offchain/authority-claimer/src/config/cli.rs

```diff
@@ -10,20 +10,17 @@ use log::{LogConfig, LogEnvCliConfig};
 use rollups_events::{BrokerCLIConfig, BrokerConfig};
```

Comment:
We should add a change in CHANGELOG mentioning this new behaviour.

Reply:
For instance, `- Added X, Y, and Z env vars to dispatcher and authority claimer. These are optional and will overwrite the values from the rollups deployment file.`